schneiderWallbox/integrationpluginschneiderWallbox.json

```diff
@@ -0,0 +1,206 @@
+                                "EVSE ready",
+                                "EV ready",
+                                "Charging",
+                                "EV ready ventilation required",
+                                "Charging ventilation required",
+                                "Stop charging",
+                                "Alarm",
+                                "Short circuit",
+                                "Digital com by EVSE state"
+                            ],
+                            "defaultValue": "EVSE not available",
+                            "displayNameEvent": "CPW state changed",
+                            "cached": false
+                        },
+                        {
+                            "id": "336da4ba-e449-418e-aee9-2c9f57d55632",
+                            "name": "pluggedIn",
+                            "displayName": "Car plugged in",
+                            "displayNameEvent": "Car plugged/unplugged",
+                            "type": "bool",
+                            "defaultValue": false,
+                            "cached": false
+                        },
+                        {
+                            "id": "0c78af0d-4775-40d8-be25-a6754ceed0f4",
+                            "name": "charging",
+                            "displayName": "Charging",
+                            "displayNameEvent": "Charging changed",
+                            "type": "bool",
+                            "defaultValue": false,
+                            "cached": false
+                        },
+                        {
+                            "id": "f9c85ef1-d44b-4169-9cd4-96ba91f1d288",
+                            "name": "maxChargingCurrent",
+                            "displayName": "Maximal charging current",
+                            "displayNameEvent": "Maximal charging current changed",
+                            "displayNameAction": "Set maximal charging current",
+                            "type": "uint",
+                            "unit": "Ampere",
+                            "defaultValue": 6,
+                            "minValue": 6,
+                            "maxValue": 32,
+                            "writable": true,
+                            "suggestLogging": true,
+                            "cached": false
+                        },
+                        {
+                            "id": "e7061cb3-27f3-4981-8672-2e4f16e87a43",
+                            "name": "totalEnergyConsumed",
+                            "displayName": "Total energy consumed",
+                            "displayNameEvent": "Total energy consumption changed",
+                            "type": "double",
+                            "unit": "KiloWattHour",
+                            "defaultValue": 0,
+                            "suggestLogging": true,
+                            "cached": false
```

Comment:
Oh, instead, you probably want to disable logging on it (note that the suggestLogging) will act on the generic log DB. It will still be logged by nymea-experience-plugin-energy in a nice and sampled way, regardless of what you set here.